src/views/partials/blogs/single.hbs

```diff
@@ -1,8 +1,13 @@
-<article class="pb-4 mb-4">
-  <img src="{{ imagelink }}" alt="{{ title }}" class="img-fluid w-100 mb-4">
-  <div class="px-4">
-    <h1>{{ title }}</h1>
-    <p class="text-muted"><small>{{ author }}, {{ date }}</small></p>
-    {{{ content }}}
+<div class="d-flex justify-content-center">
+  <div class="col-xs-12 col-md-10 mt-3">
+    <article class="mb-5">
+      <img src="{{ imagelink }}" alt="{{ title }}" class="img-fluid w-100 mb-5"
+        style="max-height: 16rem; object-fit: cover;">
+      <div class="mx-4">
+        <h1 style="overflow-wrap: normal">{{ title }}</h1>
+        <p class="text-muted"><small>{{ author }}, {{ date }}</small></p>
+        {{{ content }}}
+      </div>
+    </article>
   </div>
-</article>
+</div>
```

Comment:
And here have a wrapper hell again 🤣 
Can we merge the div on the second line with the article on the third? We'll need to add a class of mb-5 and that might do it. And of course, I'd rather prefer keeping `article` instead of `div` as a tag for semantic reasons :)

Reply:
Done :slightly_smiling_face: